cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -634,3 +636,76 @@ def _revert():
 
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(2)
+@pytest.mark.parametrize(
+    "verbosity",
+    [
+        "1 (Verbose)",
+        "2 (More Verbose)",
+        "3 (Debug)",
+        "4 (Connection Debug)",
+        "5 (WinRM Debug)",
+        "0 (Normal)",
+    ],
+)
+@pytest.mark.meta(automates=[BZ(1460788)])
+def test_service_ansible_verbosity(
+    appliance,
+    request,
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_request,
+    ansible_service,
+    verbosity,
+):
+    """Check if the different Verbosity levels can be applied to service and
+    monitor the std out
+
+    Bugzilla:
+        1460788
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    pattern = '"verbosity"=>{}'.format(verbosity[0])
```

Comment:
**Required** Please add a comment here on why you are grabbing index 0

Reply:
Simple logic:
```
In [1]: verbosity = "0 (Normal)"

In [2]: print('"verbosity"=>{}'.format(verbosity[0]))
"verbosity"=>0

```

Reply:
should I add above lines there .? What do you think .?

Reply:
Just a simple comment why you are grabbing the index so if someone takes over this FA, it is clear to them.

Reply:
Adding index 0 which will give pattern for e.g. `pattern = "verbosity"=>0`, Doable .? @jawatts

Reply:
:+1: that be a perfect comment

Reply:
Okay